src/Model/CategoryManager.php

```diff
@@ -54,7 +54,7 @@ public function getRootCategories($loadChildren = true): iterable
     /**
      * {@inheritdoc}
      */
-    public function find($categoryId): iterable
+    public function find($categoryId):  ?CategoryInterface
```

Comment:
Please remove the extra space and the build will be green.
```suggestion
    public function find($categoryId): ?CategoryInterface
```

Reply:
Sorry. I just push a new version